packages/core/src/components/button/button.tsx

```diff
@@ -252,55 +201,38 @@ function getClassList(buttonType: string, type: string, mode: string): string[]
   ];
 }
 
-/**
- * Get the classes for the color
- */
-function getColorClassList(color: string, buttonType: string, style: string, mode: string): string[] {
-  style = (buttonType !== 'bar-button' && style === 'solid') ? 'default' : style;
+function getColorClassList(buttonType: string, color: string, fill: string, mode: string): string[] {
+  let className = buttonType;
 
-  let className =
-    buttonType +
-    ((style && style !== 'default') ?
-      '-' + style.toLowerCase() :
-      '');
+  if (buttonType !== 'bar-button' && fill === 'solid') {
+    fill = 'default';
+  }
+
+  if (fill && fill !== 'default') {
```

Comment:
Same with this - are we supporting `'default'` as a fill, shouldn't it just be `'solid'`?